code/game/objects/items/devices/airlock_painter.dm

```diff
@@ -0,0 +1,61 @@
+// Airlock painter
+
+/obj/item/airlock_painter
+	name = "airlock painter"
+	desc = "An advanced autopainter preprogrammed with several paintjobs for airlocks. Use it on a completed airlock to change its paintjob."
+	icon = 'icons/obj/device.dmi'
+	icon_state = "airlock_painter"
+	item_state = "airlock_painter"
+
+	usesound = 'sound/effects/spray2.ogg'
+
+	w_class = WEIGHT_CLASS_SMALL
+	slot_flags = SLOT_BELT
+
+	materials = list(MAT_METAL = 3000, MAT_GLASS = 1000)
+
+
+/obj/item/airlock_painter/Initialize()
+	. = ..()
+
+//Only call this if you are certain that the painter will be used right after this check!
+/obj/item/airlock_painter/proc/paint(mob/user)
+	playsound(loc, usesound, 50, TRUE)
+	return TRUE
+
+/obj/item/airlock_painter/suicide_act(mob/user)
+
+	var/obj/item/organ/internal/lungs/L = user.get_organ_slot("lungs")
+	var/lungs_name = "\improper[L.name]"
```

Comment:
just [L] should do here

Reply:
Don't even need the var then

Reply:
I had some problems with it saying "X throws up his the lungs", but doing this stopped that issue.